src/api/c/cvc5.cpp

```diff
@@ -566,6 +612,22 @@ cvc5_term_t* Cvc5TermManager::copy(cvc5_term_t* term)
   return term;
 }
 
+void Cvc5TermManager::release(cvc5_op_t* op)
+{
```

Comment:
Shouldn't this be guarded with API checks?

Reply:
I'll guard this with an `if` here and will use API guards in the corresponding API functions (not all of them are implemented yet), i.e., in `cvc5_op_release()`

Reply:
I added the corresponding API functions for which the infrastructure is already in place.